pom.xml

```diff
@@ -29,7 +29,6 @@
     <logback-classic.version>1.1.2</logback-classic.version>
     <generate-config-docs-phase>prepare-package</generate-config-docs-phase>
     <hsqldb.version>2.3.2</hsqldb.version>
-    <metrics.version>3.1.2</metrics.version>
```

Comment:
Because it caused errors in modules depending on neo4j-metrics. I have no idea why, but mvn had problems with those!!